src/storage-client/src/types/sources.rs

```diff
@@ -2188,7 +2194,7 @@ impl LoadGenerator {
                 }
                 DataEncodingInner::RowCodec(desc)
             }
-            LoadGenerator::Counter => DataEncodingInner::RowCodec(
+            LoadGenerator::Counter { max_cardinality: _ } => DataEncodingInner::RowCodec(
```

Comment:
```suggestion
            LoadGenerator::Counter { .. } => DataEncodingInner::RowCodec(
```

Reply:
Could this and below be `..`? I don't care much, but it would match existing code if so.